sdks/java/maven-archetypes/examples-java8/pom.xml

```diff
@@ -43,9 +43,36 @@
       </extension>
     </extensions>
 
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+        <includes>
+          <include>archetype-resources/pom.xml</include>
+        </includes>
+      </resource>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>false</filtering>
+        <excludes>
+          <exclude>archetype-resources/pom.xml</exclude>
+        </excludes>
+      </resource>
+    </resources>
+
     <pluginManagement>
       <plugins>
         <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+          <version>3.0.2</version>
+          <configuration>
+            <delimiters>
```

Comment:
I was so eager to try this elsewhere, so I happened to discover a bug here. You have to set `useDefaultDelimiters` to `false`, otherwise `starter` archetype will fail its test. (this is a poor design on the configuration, but it is what it is)